### PR TITLE
restore compose v1 behavior

### LINKED ISSUE #8503: Output of "docker compose config" contains full path to Dockerfile
**Description**

The full path to the Dockerfile is output in the `docker compose config` output, whereas the python version `docker-compose` output a context-relative path.

The change is breaking some tooling we wrap compose with.

**Steps to reproduce the issue:**
1. docker-compose.yml:
```
version: '3.7'
services:
  example:
    build:
      context: context-path/one/
      dockerfile: Dockerfile-example
```
context-path/one/Dockerfile-example:
```
FROM scratch
```
2. `docker compose config`

**Describe the results you received:**

Output of `docker compose config`:
```
services:
  example:
    build:
      context: /Users/kevans/compose-cli-example/context-path/one
      dockerfile: /Users/kevans/compose-cli-example/context-path/one/Dockerfile-example
    networks:
      default: null
networks:
  default:
    name: compose-cli-example_default
```

**Describe the results you expected:**
Output of `docker compose config`:
```
services:
  example:
    build:
      context: /Users/kevans/compose-cli-example/context-path/one
      dockerfile: Dockerfile-example
    networks:
      default: null
networks:
  default:
    name: compose-cli-example_default
```

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker-compose --version`:**

```
docker-compose version
Docker Compose version v2.0.0-rc.2
```

**Output of `docker version`:**

```
docker version
Client:
 Cloud integration: 1.0.17
 Version:           20.10.8
 API version:       1.41
 Go version:        go1.16.6
 Git commit:        3967b7d
 Built:             Fri Jul 30 19:55:20 2021
 OS/Arch:           darwin/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.8
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.6
  Git commit:       75249d8
  Built:            Fri Jul 30 19:52:10 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.9
  GitCommit:        e25210fe30a0a703442421b0f60afac609f950a3
 runc:
  Version:          1.0.1
  GitCommit:        v1.0.1-0-g4144b63
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

**Output of `docker context show`:**  
You can also run `docker context inspect context-name` to give us more details but don't forget to remove sensitive content.

```
docker context show
default
```

**Output of `docker info`:**

```
docker info
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Build with BuildKit (Docker Inc., v0.6.1-docker)
  compose: Docker Compose (Docker Inc., v2.0.0-rc.2)
  scan: Docker Scan (Docker Inc., v0.8.0)

Server:
 Containers: 0
  Running: 0
  Paused: 0
  Stopped: 0
 Images: 0
 Server Version: 20.10.8
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: e25210fe30a0a703442421b0f60afac609f950a3
 runc version: v1.0.1-0-g4144b63
 init version: de40ad0
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 5.10.47-linuxkit
 Operating System: Docker Desktop
 OSType: linux
 Architecture: x86_64
 CPUs: 2
 Total Memory: 1.94GiB
 Name: docker-desktop
 ID: <REDACTED>
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 HTTP Proxy: http.docker.internal:3128
 HTTPS Proxy: http.docker.internal:3128
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

**Additional environment details (AWS ECS, Azure ACI, local, etc.):**
local

### PR DESCRIPTION
**Pull request by ndeloof**
Resolves #8503

### CODE REVIEW

File: cmd/compose/convert.go [hunk 1]
```diff
@@ -90,7 +91,8 @@ func convertCommand(p *projectOptions, backend api.Service) *cobra.Command {
 	}
 	flags := cmd.Flags()
 	flags.StringVar(&opts.Format, "format", "yaml", "Format the output. Values: [yaml | json]")
-	flags.BoolVar(&opts.resolve, "resolve-image-digests", false, "Pin image tags to digests.")
+	flags.BoolVar(&opts.resolveImageDigests, "resolve-image-digests", false, "Pin image tags to digests.")
+	flags.BoolVar(&opts.resolvePaths, "resolve-paths", false, "Resolve relative paths")
```
Comment (mat007) [inline cmd/compose/convert.go:95]: nit: I’m not sure based on the description it’s super obvious what it does 🤔 
(also descriptions seem to end with a `.`)

Comment (mat007) [reply cmd/compose/convert.go:95]: I mean: why and when would a user want to enable this?

Comment (ndeloof) [reply cmd/compose/convert.go:95]: any better suggestion for flag description?

Comment (mat007) [reply cmd/compose/convert.go:95]: Maybe we don’t really need this as a flag, actually?

Comment (ndeloof) [reply cmd/compose/convert.go:95]: Sure, this is not actually a requirement to expose this to the end-user.
I just used this in the past to help diagnose relative path issues, but that's only me :P

**Review by mat007 — APPROVED** [review]
Makes sense!

**Pull request merged**